src/device-registry/utils/create-activity.js

```diff
@@ -146,7 +146,7 @@ const createActivity = {
       } else if (responseFromDeviceSearchCheck.success === false) {
         const filter = { _id: ObjectId(site_id) };
         const responseFromListSite = await createSiteUtil.list({
```

Comment:
Hi @BenjaminSsempala , I think `util/create-site.list` function expects a request parameter as the only input.
See attached screenshot.
![Uploading Screenshot 2023-10-11 at 21.33.19.png…]()